Source/DafnyDriver/Commands/VerifyCommand.cs

```diff
@@ -76,6 +92,18 @@ public static async Task<int> HandleVerification(DafnyOptions options) {
       await verificationSummarized;
       await verificationResultsLogged;
       await proofDependenciesReported;
+
+      if (!resolution.HasErrors && options.BoogieExtractionTargetFile != null) {
+        using (var engine = ExecutionEngine.CreateWithoutSharedCache(options)) {
+          try {
+            var extractedProgram = BoogieExtractor.Extract(resolution.ResolvedProgram);
+            engine.PrintBplFile(options.BoogieExtractionTargetFile, extractedProgram, true, pretty: true);
+          } catch (ExtractorError extractorError) {
+            options.OutputWriter.WriteLine($"Boogie axiom extraction error: {extractorError.Message}");
```

Comment:
Could you resolve these warnings?

Reply:
Are you saying to remove this `try catch`? These errors are generated if the input to the extractor is invalid, so they need to stay.

Reply:
No, the build was outputting warnings because this was using `WriteLine` instead of `WriteLineAsync`